src/index.ts

```diff
@@ -1,14 +1,40 @@
-import express from "express";
-const app = express();
-const port = 8080; // default port to listen
+import { fetchOrderbook } from "./fetch_orderbook"
+import {
+  Orderbook,
+  transitiveOrderbook
+} from "@gnosis.pm/dex-contracts/src/orderbook"
+import express from "express"
 
-// define a route handler for the default home page
-app.get("/", (req, res) => {
-  res.send("Hello world!");
-});
+const app = express()
+const port = 8080 // default port to listen
+
+// Setup orderbook fetching every 1000s
+let orderbooks: Map<string, Orderbook>
+setInterval(async () => {
+  orderbooks = await fetchOrderbook()
+}, 1000)
+
+app.get("/orderbook", (req, res) => {
+  const transitive = transitiveOrderbook(
+    orderbooks,
+    req.query.base,
+    req.query.quote,
+    2
+  )
+  res.send(JSON.stringify(transitive))
```

Comment:
nit:
```suggestion
  res.json(transitive)
```
That way the correct HTTP response headers get set